src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -622,6 +624,15 @@ export class AutoColumnSize extends BasePlugin {
     privatePool.get(this).cachedColumnHeaders = this.hot.getColHeader();
   }
 
+  onAfterFormulasValuesUpdate(changes) {
+    const filteredChanges = arrayFilter(changes, change => change.address?.col !== undefined ?? false);
```

Comment:
Is `change.address?.col !== undefined ?? false` different from just `change.address?.col !== undefined`? (Or our helper's `isDefined(change.address?.col)`?)

Reply:
`isDefined(change.address?.col)` will be the best choose

Reply:
done in eb94c0a